test/ingress/conformance/tests/httproute-http2rpc.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.higress.io/v1
+kind: Http2Rpc
+metadata:
+  name: httproute-http2rpc-demo
+  namespace: higress-system
+spec:
+  routeType: "DUBBO"
+  routeService: "com.dubbo.demo.api.DemoService"
+  routeServiceVersion: "1.0.0"
+  methods: 
+  - serviceMethod: "sayHello"
+    headersAttach: "*"
+    httpMethods: 
+    - "ALL_GET"
```

Comment:
Replace with GET, POST, etc.

Reply:
OK, let me change it to make it User friendliness.